projects/approved-premises-and-oasys/deploy/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: '1.0'
```

Comment:
According to https://helm.sh/docs/topics/charts/#the-appversion-field, this field corresponds to the version of the app, so I think doing the `sed`/`yq` to set it would be a good idea.  What do you think? 

Also the other `version` field is the chart version, and should follow semver, so maybe we just leave it as 1.0.0 given we won't really be changing this file - i.e. remove the `--version` flag in the deployment ?